aeon/classification/dictionary_based/_weasel_v2.py

```diff
@@ -68,6 +68,14 @@ class WEASEL_V2(BaseClassifier):
         If the array contains True, words are computed over first order differences.
         If the array contains False, words are computed over the raw time series.
         If both are set, words are computed for both.
+    support_probabilities : bool, default = False
+         If set to False, a RidgeClassifierCV will be trained, which has higher accuracy
+         and is faster, yet does not support predict_proba.
+         If set to True, a LogisticRegression will be trained, which does support
+         predict_proba(), yet is slower and typically less accurate. predict_proba() is
+         needed for example in Early-Classification like TEASER.
+
+         Deprecated and will be removed in v0.6.2.
```

Comment:
```suggestion
    support_probabilities : str or bool, default="deprecated"
         Old parameter to support probabilities, no longer used as probabilities 
         are supported by default .
         
         Deprecated and will be removed in v0.8.0.
```

It no longer does any of the documented stuff when changing, so better just to say its useless IMO.

Reply:
Thanks, I have updated the text.